app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,86 @@
+<main>
+  <section class="spotlight">
+    <h2 class="spotlight__header">
+      <span class="spotlight__header--prefix">
+         "Media Spotlight" *::after
+      </span>
+      <a class="spotlight__link-to" href=""><%=Work.spotlight.title%>by<%=Work.spotlight.creator%></a>
+    </h2>
+
+  <p class="spotlight__description"><%=Work.spotlight.votes.length%> votes- <%=Work.spotlight.description%></p>
+  </section>
+
+  
+    
+   <hr class="root__hr">  
+   <section class="top-ten__container">
+       <section class="top-ten__list-container">
+       <%top_movies = Work.top_ten("movie")%>
+        <h2 class="top-ten__header"> Top Movies
+      <h2> 
```

Comment:
You accidentally opened a new `<h2>` here instead of closing the last one:

```suggestion
        <h2 class="top-ten__header"> Top Movies
      </h2> 
```

Also, your indentation is wonky here.